Cargo.toml

```diff
@@ -18,6 +18,8 @@ crate-type = ["rlib"]
 doctest = false
 
 [dependencies]
+macro-trace-object = { path = "macros/trace_object"}
```

Comment:
Sure. I will add a prefix. But I would prefer putting each macro to a seperate crate so that each crate will be straightforward and simple to understand.